packages/ui/src/components/editor/panels/Files/download/projectDownload.tsx

```diff
@@ -92,20 +105,34 @@ export const ProjectDownloadProgress = () => {
   const { t } = useTranslation()
   const downloadState = useMutableState(DownloadProjectState)
   const isDownloading = downloadState.isDownloading.value
+  const isFetching = downloadState.isFetching.value
+
   const completed = bytesToSize(downloadState.progress.value)
   const total = bytesToSize(downloadState.total.value)
   const progress = (downloadState.progress.value / downloadState.total.value) * 100
 
-  return isDownloading ? (
-    <div className="flex h-auto w-full justify-center pb-2 pt-2">
-      <div className="flex w-1/2">
-        <span className="inline-block pr-2 text-xs font-normal leading-none text-theme-primary">
-          {t('editor:layout.filebrowser.downloadingProject', { completed, total })}
-        </span>
-        <div className="basis-1/2">
-          <Progress value={progress} />
+  return (
+    <>
+      {isDownloading && (
+        <div className="flex h-auto w-full justify-center pb-2 pt-2">
+          <div className="flex w-1/2">
+            <span className="inline-block pr-2 text-xs font-normal leading-none text-theme-primary">
+              {t('editor:layout.filebrowser.downloadingProject', { completed, total })}
+            </span>
+            <div className="basis-1/2">
+              <Progress value={progress} />
+            </div>
+          </div>
         </div>
-      </div>
-    </div>
-  ) : null
+      )}
+      {isFetching && (
+        <LoadingView
+          titleClassname="mt-0"
+          containerClassName="flex-row mt-1"
+          className="mx-2 my-auto h-6 w-6"
+          title={t('editor:layout.filebrowser.fetchingProject')}
+        />
+      )}
+    </>
```

Comment:
this seems good

currently we have a lot of progress bars. we should have some sort of designs which unifies all the progress bars in the studio. 
will ask Tarek in today's studio meeting

Reply:
sure review it i the context of the PR, and merge it in, we can work on the unification later, this is a priority PR